src/ngraph/runtime/gpu/gpu_memory_manager.cpp

```diff
@@ -26,7 +26,7 @@ constexpr const uint32_t initial_buffer_size = 10 * 1024 * 1024;
 
 runtime::gpu::GPUMemoryManager::GPUMemoryManager(GPUPrimitiveEmitter* emitter)
     : m_buffer_offset(0)
-    , m_buffered_mem(initial_buffer_size)
+    , m_buffered_mem(initial_buffer_size, 0)
```

Comment:
Why zero initialize?

Reply:
good question, it's not necessary to zero initialize it. but it's a good practice to zero initialize. It's by default for array in c++.

Reply:
I should not say by default, if you do like this, it will be zero initialized.
```
T () ;
T t = {} ; 
T {} ;
```